helm-charts/yunikorn/templates/deployment.yaml

```diff
@@ -83,6 +83,10 @@ spec:
               value: "{{ .Values.UserLabelKey }}"
             - name: OPERATOR_PLUGINS
               value: "{{ .Values.operatorPlugins }}"
+            {{- if .Values.placeholderImage }}
```

Comment:
Can we remove the `if` block and instead add a default image to `values.yaml`?

Reply:
I agree, this is cleaner.